gmail/gmail_types.bal

```diff
@@ -344,12 +229,12 @@ public type MessagePartHeader {
 documentation{
     Represents GMail error
 
-    F{{errorMessage}} - GMail error message
+    F{{message}} - GMail error message
     F{{cause}} - The error which caused the GMail error
     F{{statusCode}} - The error status code
 }
 public type GMailError {
-    string errorMessage;
+    string message;
     int statusCode;
     error? cause;
```

Comment:
The error type has to be in the following format for the ballerina runtime to identify it as an error. Shall we change it?
```ballerina
public type GMailError {
    string message;
    error? cause;
    int statusCode;
}
```

Reply:
Changed the custom error type with respect to **error** type